app/controllers/well_known_controller.rb

```diff
@@ -1,8 +1,15 @@
 # frozen_string_literal: true
 
+require_relative '../../lib/health_cards'
+
 # WellKnownController exposes the .well-known configuration to identify relevant urls and server capabilities
 class WellKnownController < ApplicationController
   def index
     render json: Rails.configuration.well_known.to_json
   end
+
+  def jwks
+    @issuer = HealthCards::Issuer.new ::Configuration.key_path
```

Comment:
We need to figure out a consistent way to do our config loading. My preference would be to put things in the existing Rails app config (see index method) rather than creating our own config object.

Reply:
@Jammjammjamm had me create the `Configuration` module to make it easier for test and development. The health_cards library also uses this `Configuration` module. Any thoughts Steve?

Reply:
The health cards library should not be using the `Configuration` module. I don't have strong feeling over using this or rails configuration, as long as the environment variable references are contained in a single place.

Reply:
Actually now that I look at it again, the `Configuration` module is only used in the issuer tests but we can discuss how we want to do config moving forward.